app/styles/libs/_helpers.scss

```diff
@@ -55,7 +55,7 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 
 [data-ember-action] {
   &:not(:disabled) {
-    cursor: pointer;
+    cursor: text;
```

Comment:
Wrong.
This previous css property wis a workaround that makes an HTML element tied with an `action` helper clickable which is initially not clickable.
For example: In various places we have a `div` tied to an `action` helper. Now, after this change, that `div` will no longer be clickable. 
This is a very low priority bug. We cannot block other places in order to solve this. I am closing it therefore.